api/src/user/user.controller.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpException,
+  Param,
+  ParseIntPipe,
+  Post,
+  Put,
+  Query,
+} from '@nestjs/common';
+import { Prisma } from '@prisma/client';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @Get()
+  async getUsers(
+    @Query('filter') filter: string,
+    @Query('sort') sort: string,
+    @Query('range') range: string,
+  ) {
+    let params;
+    const filters = JSON.parse(filter);
```

Comment:
If a uses passes an incorrect filter string, this will throw. I guess the framework will return a 500, while the error should really be a 4xx in this case